### DISCUSSION TITLE
脚本进入死循环

### DISCUSSION THREAD

**Discussion by EunoiaCody** [monlor/docker-xiaoya#235] [Q&A]
程序运行中，log不断出现以下内容

metadata-1  | alist启动完成，等待10秒后开始下载元数据...
metadata-1  | 开始下载元数据，如果有问题无法解决，请删除目录 /media/temp 下的所有文件重新启动.
metadata-1  | Emby metadata has been downloaded. Delete the file /media/config/emby_meta_finished to re-extract.
metadata-1  | Cleaning up Emby media...
metadata-1  | 目录/media/temp最少需要剩余空间：60G，目前仅剩：43G

我准备了一个512G的硬盘，才用了不到一半，不应会出现空间不足的提示才对，且我进入/media目录中，并没有看见temp文件夹，当我试图创建一个的时候，刚创建好就消失了，于是我退出进程再次创建，然而程序运行时，此文件夹再次消失且再次出现空间不足的提示（如上），目前alist已经能够运行

电脑是M2的Mac mini，数据保存在硬盘（服务端部署目录），没有对compose和env进行修改

**Comment by EunoiaCody** [comment]
这似乎属于bug，所以我会关闭这个discussion，提一个issue

**Comment by ywlhlp** [comment]
貌似作者好长时间没有解决问题了，也不知道是不是放弃了这个项目

**Reply by EunoiaCody** [reply]
应该没有吧，我看在issue里，作者还是很活跃的
